src/client/Microsoft.Identity.Client/ManagedIdentity/V2/ImdsV2ManagedIdentitySource.cs

```diff
@@ -309,5 +353,92 @@ private static string ImdsV2QueryParamsHelper(RequestContext requestContext)
 
             return queryParams;
         }
+
+        private async Task<string> GetAttestationJwtAsync(
```

Comment:
please add detailed xml comment for this function

Reply:
We usually don't for internals